config/jsdoc.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
         description: 'Operations for data science service',
       },
       {
+        // 'courses'?
```

Comment:
@paulstgermain we were a little unclear as to what this file does, so we didn't change the naming here. Please advise, thanks!

Reply:
Seconded!